### ISSUE TITLE
ConvertFrom-ExcelToSqlInsert 7.8.4 still does not escape single quotes

### ISSUE THREAD

**Issue by SarSerg** [dfinke/ImportExcel#1412]
This is a follow-up of #525
I've installed and updated ImportExcel module, now in 7.8.4 version.
I have the same error cited in #525: long strings with embedded quote keep the single quote, and the INSERT fails.

This is a sample of VALUE values, look at the string "Peter's test":

Values('f19a7280-6e87-ed11-959e-00155d701c7b', '281db251-8887-ed11-b2a5-00155d70225c', '0', NULL, '0', '0', '0', '0', NULL, '0', '0', NULL, '0', '0', 'Commited (hard) booking type', '0', 'Commited', '0', '0', '0', '800', '800', '12/29/2022 14:52:28', '2', '0', '0', '01/10/2023 17:00:00', '0', 'False', 'True', '0', '0', '01/04/2023 13:54:01', '23', '0', '0', '100', '0', '800', '16', '800', '0', '0', '800', '16', '16', '0', '20', '01/09/2023 08:00:00', '0', '0', '0', NULL, '0', '16', '16', 'True', '**Peter's test**', '1e4dd7fa-7687-ed11-b1ab-00155d706b3b', 'Client Solution Architect', '75463135-8887-ed11-b2a5-00155d70225c', 'True', 'Prepare workshop 2', '000047df-f395-462a-a1f2-7011fef3a122', 'Assignment type which indicates a normal assignment that has a resource assigned. This assignment type is copied from the Published Database.', 'Normal Assignment', NULL, NULL, NULL, NULL, NULL, NULL);

Is there a way to force double quotes as strings delimiters or the double single quotes ('') instead than single quote inside strings?

Thanks

**Comment by dfinke** [comment]
I don't believe there is.

Need to take a look when I can. May provide a `$Delimiter` parameter.

**Comment by sannae** [comment]
Hi! First time contributor here, please be understanding... 😅 

I'd like to work on this issue, I don't know if I can once it's been already assigned...

Apparently the fix was committed to master in PR #732 solving issues #512 and #525, but somehow was lost (as tracked by #920, closed automatically as stale), I believe when converting the old `/ConvertFromExcelToSqlInsert.ps1` to the new `/Public/ConvertFrom-ExcelToSqlInsert.ps1`.

Probably it's just a matter of adding the following to the [`$values`](https://github.com/dfinke/ImportExcel/blob/ab4d03c984fc7a8b129c8fe03af02b1f29f743e5/Public/ConvertFrom-ExcelToSQLInsert.ps1#L45) variable assignment:
```powershell
"'" + (($record.$propertyName) -replace "'","''") + "'" # Escaping any single quote within the $record.$propertyName, while keeping the starting and ending single quotes
```

Thanks

**Comment by dfinke** [comment]
Thanks for checking that out. Sounds good for you taking it on. The only thing is. Adding pester tests. I haven’t looked, don’t remember if there are tests for this function. Having one for this change and a few others would be optimal. 

Also would be easier to review. Feel free to submit a PR.

**Comment by sannae** [comment]
Ok thank you! I will add some Pester tests if there is none 👍🏻

**Comment by sannae** [comment]
Ok! Then we'll keep it as it is? Maybe I'll just 
- Add the parameter to the cmdlet documentation (in folder `mdHelp`)
- Add Pester tests to check the parameter
What do you think?

**Comment by dfinke** [comment]
Perfect!

I'll add it to the changelog once it is merged.

Thank you

**Comment by dfinke** [comment]
@sannae Please make that a separate issue. Thanks!
